packages/blaze-html-templates/package.js

```diff
@@ -1,14 +1,14 @@
 Package.describe({
   name: 'blaze-html-templates',
   summary: "Compile HTML templates into reactive UI with Meteor Blaze",
-  version: '1.1.0',
+  version: '1.1.0-1-beta.2',
```

Comment:
Ehm. Patch version change is: `1.1.0` -> `1.1.1`.  At least according to semver.org.